app/views/hyrax/base/_workflow_actions.html.erb

```diff
@@ -12,7 +12,7 @@
         <% presenter.workflow.actions.each do |key, label| %>
           <div class="radio">
             <label>
-              <input type="radio" name="workflow_action[name]" id="workflow_action_name_<%= key %>" value="<%= key %>">
+              <%= radio_button_tag 'workflow_action[name]', key, :checked => !!key == 'comment_only' %>
```

Comment:
`!!key` is going to cast whatever key is to a boolean, right?  then compare that with the string `'comment_only'` which will always be false? Am I reading that right?

Reply:
Yup, you're right - DOH - I'll rework that logic.